doc/nrf/gs_installing.rst

```diff
@@ -219,6 +219,51 @@ Therefore, remember to regularly check for updates:
          pip3 install -U west
 
 
+Installing additional toolchains for Connected Home over IP applications development
+====================================================================================
+
+.. note::
+   If you are not interested in the `Connected Home over IP`_ applications, you can ommit steps presented in this section.
+
+`Connected Home over IP`_ project is using `GN`_ meta-build system to generate files for `Ninja`_ and both tools are necessary in order to build an applications.
+However Ninja is also required tool for Zephyr, so it is assumed that it should be installed with the rest of Zephyr's dependencies and GN is the only missing |NCS| requirement.
+
+.. note::
```

Comment:
Should we move the note right below the previous one ("If you are not interested in...")?

Reply:
I don't have strong statement on this, as text is quite short, so second note is not hidden somewhere deep inside the section. I can change it, if you find it useful. Would you prefer to have two notes placed one by one, or both information in the same note?

Reply:
Yes, I think it's better if such notes are at the beginning so that one knows in advance if it's even useful to read the text for him/her.